### ISSUE TITLE
SenseBox (research)

### ISSUE THREAD

**Issue by Danaal** [Danaal/Open_Science_Project#8]

**Comment by Danaal** [comment]
**SenseBox:** is a project that makes it easy for users especially in digital education, citizen of science, professional data collectors to program microcontrollers (mini computers) and work with them. 
1- **Accessibility** [3]: the products they are offering are not available to be delivered to all countries just for around 20 and they are with costs, so they are not accessible for people who can’t pay the fees.
2- **Discoverability[**4]: the project has a present in all social media and many projects in github. They have multiple workshops, but there is not a lot of information regarding locations and timing (contact the support team). Therefore, it would be a bit difficult for the public to contact the team (some effort needed).
3- **Publications** [4]: There are multiple publications found on the internet regarding the sensebox system.
4- **Reusability** [3]: The website provides documentation on the hardware and software usage of sensebox and the system itself is built on top of the idea of Arduinos and many other microprocessors.
5- **Reproducibility** [2] The project doesn’t provide information regarding the process of reproducing the products or the softwares they created.
6- **Transparency**[2]: There is not a lot of info regarding the rating of the product or customer interaction with it unless it wasn’t visible to me.
7- **understandability** [5]: the project precisely indicates that the product can be for different users in education or at homes and it has clear documentation for the usage of the tool boxes.
8- **collaboration**[4]: the project has a good sponsor network and it reaches for different stakeholders in education and home innovation.
